addons/web/static/src/views/form/form_controller.js

```diff
@@ -317,6 +336,7 @@ export class FormController extends Component {
     }
 
     async onPagerUpdate({ offset, resIds }) {
+        this.duplicateId = undefined;
```

Comment:
that's weird, if I understand correctly, if you duplicate a record, make a change on a translatable field then do page next, you won't update translations

Reply:
maybe you should reset duplicateId to undefined in the hook onWillLoadRoot ?